helpers/voting/getVoteMetaData.ts

```diff
@@ -21,9 +21,10 @@ export function getVoteMetaData(
   decodedAncillaryData: string,
   umipDataFromContentful: ContentfulDataT | undefined
 ): VoteMetaDataT {
-  const isAssertion = ["assertionId:", "ooAsserter:", "childChainId:"].every(
-    (lookup) => decodedAncillaryData.includes(lookup)
```

Comment:
this "childChainId" check was causing us to miss that this was an assertion. it doesnt seem to be included in the ancillary data for this vote. figured this is still strict enough to not have false positives